### ISSUE TITLE
Netgear firmware unable to simulate in the firmadyne

### ISSUE THREAD

**Issue by susantaroy2002** [firmadyne/firmadyne#78]
Unable to simulate Netgear firmware, not if there any thing wrong on .chk  file extension. 

Here I am sharing some details about my blocker, 
1. Tried on 2 netgear firmware version - "D8500-V1.0.3.39_1.0.1.chk" and "R7000-V1.0.9.28_10.2.32.chk"
2. attaching the error screenshot. 

Please let me know if you need any further details to investigate this issue.

<img width="1326" alt="screen shot 2018-06-03 at 12 20 32 pm" src="https://user-images.githubusercontent.com/26112385/41102437-8401ff92-6a84-11e8-93e8-9f33f6b59f15.png">
<img width="1322" alt="screen shot 2018-06-03 at 12 21 25 pm" src="https://user-images.githubusercontent.com/26112385/41102439-8437aed0-6a84-11e8-8ea6-52b130510cf3.png">
<img width="1323" alt="screen shot 2018-06-03 at 12 24 13 pm" src="https://user-images.githubusercontent.com/26112385/41102441-84694eea-6a84-11e8-967e-fb03373ea962.png">

**Comment by susantaroy2002** [comment]
Yeah, It work directly with firmadyne, however tested with dlink it works. But not sure about the NetGear. Any SME that you can redirect me.

**Comment by ddcc** [comment]
I don't know what an SME is.

**Comment by santhoshn16** [comment]
try dropping the db, user, create them again and follow the steps you will get it.
